.azure/gpu-tests-pytorch.yml

```diff
@@ -112,7 +112,7 @@ jobs:
 
     - bash: |
         set -e
-        pip install -r requirements/pytorch/strategies.txt --find-links ${TORCH_URL}
+        pip install -r requirements/pytorch/strategies.txt --find-links ${TORCH_URL} --upgrade-strategy eager
```

Comment:
This did not work. If you see the job env details, you'll see it didn't upgrade it. It's missing the -U flag.

And either way it won't work, because eager mode also upgrades the dependencies of our dependencies etc. So it creates version conflicts unless you install all dependencies in a single `pip install ...` command so that the resolution algorithm can find a valid resolution. But our CI installs dependencies in steps.